# INFERENTZIA ESTATISTIKOA: Hirugarren praktika

## Hipotesi kontrasteak: Populazioa bakarra

Aurreko praktikarekin (estimazioa eta KTak) harreman handiko praktika da hau. Izan ere, praktika horretan erabili genituen komandoak erabiliko ditugu eta ikusiko dugu nola aldatu behar diren hipotesi kontrasteak burutzeko.


In [2]:
# txt datuak irakurri

Datuak <-read.table(file="altuerak.txt", header=TRUE, dec=",", sep="\t")

# Hurrengo paketea instalatu eta kargatu behar duzue 
# install.packages("EnvStats")
# library(EnvStats)


### Populazioaren pisuaren batezbestekoa. 

### Hurrengo aldebakarreko HK planteatzen dugu
* $H_0:\ \mu = 80$

  $H_1:\ \mu \neq 80$

In [3]:
t.test(Datuak$PISUA,alternative='two.sided',mu=80,conf.level=.99)


Results of Hypothesis Test
--------------------------

Null Hypothesis:                 mean = 80

Alternative Hypothesis:          True mean is not equal to 80

Test Name:                       One Sample t-test

Estimated Parameter(s):          mean of x = 72.54386

Data:                            Datuak$PISUA

Test Statistic:                  t = -5.44299

Test Statistic Parameter:        df = 170

P-value:                         1.805397e-07

99% Confidence Interval:         LCL = 68.97529
                                 UCL = 76.11243


#### Azalpena:
Lortutako estatistikoa -5.44299 da, askatasun graduak 170 eta p-balioa 1.805397e-07 (hau da, $1.8\cdot 10^{-7}$). %1eko adierazgarritasun mailaz, populazioaren batezbesteko pisua ez dela 80 ondorioztatzen dugu.

#### Oharrak: 
* 80 balioarekin kontrastatu ordez, 60 balioarekin kontrastatzeko, mu=80 jarri ordez mu=60 jarri behar da.
* Aldebateko eskuinetiko kontrastea egiteko, alternative="greater" jarri behar da ($\mu>80$ delako $H_1$)
* Aldebateko ezkerretiko kontrastea egiteko, alternative="less" jarri behar da ($\mu<80$ delako $H_1$)
* Beste adierazgarritasun mailarekin HK egiteko, conf.level aldatu behar da.

### Populazioaren altueraren bariantzaren HK 
* $H_0:\ \sigma^2=16$

  $H_1: \ \sigma^2\neq 16$

In [4]:
varTest(Datuak$ALTUERA,alternative  = "two.sided",sigma=16,conf.level=0.95)


Results of Hypothesis Test
--------------------------

Null Hypothesis:                 variance = 16

Alternative Hypothesis:          True variance is not equal to 16

Test Name:                       Chi-Squared Test on Variance

Estimated Parameter(s):          variance = 11.08242

Data:                            Datuak$ALTUERA

Test Statistic:                  Chi-Squared = 117.7507

Test Statistic Parameter:        df = 170

P-value:                         0.001614072

95% Confidence Interval:         LCL =  9.057947
                                 UCL = 13.874455


#### Azalpena:
Lortutako estatistikoaren balioa 117.7507 da, Khi karratuaren askatatasun graduak 170 eta p-balioa 0.001614 da. %5eko adierazgarritasun mailaz, altueraren bariantza ez dela 16 ondorioztatzen dugu. 

#### Oharrak: 
* 16 balioarekin kontrastatu ordez, 9 balioarekin kontrastatzeko, sigma=16 jarri ordez sigma=9 jarri behar da.
* Aldebateko eskuinetiko kontrastea egiteko, alternative="greater" jarri behar da ($\sigma>16$ delako $H_1$)
* Aldebateko ezkerretiko kontrastea egiteko, alternative="less" jarri behar da ($\sigma<16$ delako $H_1$)
* Beste adierazgarritasun mailarekin HK egiteko, conf.level aldatu behar da.

## Bizkaikoen proportzioaren HK
* $H_0:\ p\leq 0.15$

  $H_1:\ p>0.15$

In [8]:
# zenbat bizkaitar dauden zenbatzen dugu
Datuak.bizkaia <- subset(Datuak,JAIOTERRIA==1)
kopuru.bizkaia <- length(Datuak.bizkaia$ALTUERA)
#kopuru.bizkaia
guztiak <- length(Datuak$ALTUERA)
#guztiak
prop.test(kopuru.bizkaia,guztiak,p=0.15,alternative="greater",conf.level=0.95)


	1-sample proportions test with continuity correction

data:  kopuru.bizkaia out of guztiak, null probability 0.15
X-squared = 7.5736, df = 1, p-value = 0.002962
alternative hypothesis: true p is greater than 0.15
95 percent confidence interval:
 0.1771247 1.0000000
sample estimates:
        p 
0.2280702 


#### Azalpena:
Lortutako estatistikoaren balioa 7.573558 da eta p-balioa 0.002961537. Ondorioa hauxe da: %5eko adierazgarritasun mailaz, bizkaikoen proportzioa 0.15 baino handiagoa da.
 
#### Oharrak: 
* 0.15 balioarekin kontrastatu ordez, 0.4 balioarekin kontrastatzeko, p=0.15 jarri ordez p=0.4 jarri behar da.
* Aldebiko ezkerretiko kontrastea egiteko, alternative="less" jarri behar da ($p<16$ delako $H_1$).
* Aldebakarreko kontrastea egiteko, alternative="two.sided" jarri behar da 
* Beste adierazgarritasun mailarekin HK egiteko, conf.level aldatu behar da.